src/frontend/src/handler/create_source.rs

```diff
@@ -343,6 +343,78 @@ pub(crate) async fn resolve_source_schema(
                 ..Default::default()
             }
         }
+        SourceSchema::DebeziumMongoJson => {
+            if columns.is_empty() {
+                let mut col_id_gen = ColumnIdGenerator::new_initial();
+                let pk_id = col_id_gen.generate("_id");
+                columns.push(ColumnCatalog {
+                    column_desc: ColumnDesc {
+                        data_type: DataType::Varchar,
+                        column_id: pk_id,
+                        name: "_id".to_string(),
+                        field_descs: vec![],
+                        type_name: "".to_string(),
+                        generated_column: None,
+                    },
+                    is_hidden: false,
+                });
+                columns.push(ColumnCatalog {
+                    column_desc: ColumnDesc {
+                        data_type: DataType::Jsonb,
+                        column_id: col_id_gen.generate("payload"),
+                        name: "payload".to_string(),
+                        field_descs: vec![],
+                        type_name: "".to_string(),
+                        generated_column: None,
+                    },
+                    is_hidden: false,
+                });
+                pk_column_ids.push(pk_id);
+                row_id_index.take();
+            }
+
+            // return err if user has not specified a pk
+            if row_id_index.is_some() {
+                return Err(RwError::from(ProtocolError(
+                    "Primary key must be specified when creating source with row format debezium."
+                        .to_string(),
+                )));
+            }
```

Comment:
For the case below.
```
CREATE TABLE mongo_customers (
	_id BIGINT, -- NO PRIMARY KEY HERE
	payload jsonb
) ...
```

Reply:
actually, we extract `_id` to make it primary key

Reply:
If the user leaves the CREATE TABLE xx (...) blank, we will create `_id` column and make it be primary key, otherwise we will not help users to create columns.

Reply:
> If the user leaves the CREATE TABLE xx (...) blank, we will create _id column and make it be primary key 

what is the type for `_id` column?

Reply:
> The important detail, however, is that you can also pass your own _id when inserting a document and it doesn't have to be of type ObjectId. In fact, according to MongoDB [document specification](https://www.mongodb.com/docs/manual/core/document/), it can be of any type except arrays
> from: https://stackoverflow.com/questions/51322231/what-is-the-type-of-id-field-in-mongo-database

`_id` may not always be a verchar.